src/style.css

```
@@ -1 +1,65 @@
-/* Styles go here */
+body {
+  margin: 0;
+  display: block;
+  --main-color: black;
+  --active-color: #00acdc;
+}
+
+html {
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  font-family: "Roboto", sans-serif;
+}
+
+.header__container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+  letter-spacing: 0;
+  text-align: left;
+  text-transform: uppercase;
+}
+
+.navigation {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+}
+
+.nav__link {
+  display: flex;
+  position: relative;
+  align-items: center;
+  text-decoration: none;
+  color: var(--main-color);
+  height: 60px;
+}
+
+li:nth-child(n + 2) {
```

```suggestion
li:not(:first-child) {
```
but it seems not deployed yet